depot/app/views/shared/_flash.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if !flash.keys.empty? %>
```

Comment:
<% flash.each do |key, value| %>
  <p id="notice"><%= value %></p>
<% end %>

can it be written like this.?